src/plugins/utilities/layout/grid-gap.js

```diff
@@ -21,6 +21,8 @@ module.exports = function () {
     const newUtilities = {
       // Center an element horizontally.
       '.grid-gap': {
+        'grid-gap': gaps['xs'],
+        gap: gaps['xs'],
         ...output
```

Comment:
To work with removing "xs" from screens. I took the xs value out of the for loop for media query. xs doesn't need to be wrapped in a media query since we're doing mobile first.